cmd/emailreport/README.md

```diff
@@ -10,7 +10,30 @@ file can be specified by command line or it is installed in
 WARNING: The password for the SMTP authentication will be in TOML
 configuration text plane file.
 
-# Usage
+## Options
+`emailreport` takes a number of options.
```

Comment:
The `emailreport` file takes a number of options.

Reply:
emailreport is a binary I wonder if could work this "The emailreport tool takes a number"

Reply:
yes that works!